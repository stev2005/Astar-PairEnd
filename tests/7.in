GATCGGTTCAGTGGCTGATTGATTTATGCCATCTCTCAGGCAGTTTGCTCCCCAAAAAGCCGAGGCCTGTGTAAGTTAAGGCATCAACAGGCCTTTAAGGAACAATTTATGGCGCGGCATAGACCAAGAGAATGATCTTCTCTATCGCGGAGCCTGCTACGCGCTGAAGTGCACCT
15
TCCGCTTGTCTGGCATTCGCGGTCAAGGCCACGCTACGGATGGCTTAGAGTGCT
CTGTCTAGGCCGGTAGGGCACAATATTTAGTCCCCGACAGGCACACTTTATCATACGTAGCTATACCAGGTATTCAACTACGCGCC
ACTGGGGCGTTGCCCGACTACATCGCGATATTCGCTAACGTAACCGTCTGGATCGCTACGAACCGGCGGGAGGATTGGACAGGTAAATACCCCGGTATCTCCCTGAGAGTGGC
GCAGAAACCCTACATGTCAAAGATGGGAATAGCACCCCTGGGGTGGGCTGCTACGGTATAATGCATGCACTTAGGGAAATGCGGTCCGCAGCTCGTACACGA
GGGCGTCCCGTACACAGCGTATGAATACCACAGTCAGGCAACAGCCGTGATGACGAATCCTGCGCGGAAAACCATGCCGTCCCCGACGTGTGACAGTTGATGCAGAGTCAGGCTATTCCGACCAGCGCAACTGGTAGGTTGCGACGTATC
TGAAACGCGTTCCCCGATCTCTCCATCATGTGATGACCCTAAACGCTGACCGATTAGAACGAAGTGGATATTAACGCAAGCGACCTCTAGCGGCCCTTGTTCGACTGTATCCC
TGAACAGGTATAGTTAGTCATTACGAATCCAATACCCTTAATAGGTGAGTACGATAATTGCTGAATCCTCCTACCGATG
GGACTTCTTCCAAAAATGCGTGCTTGCACTTACTCATGAGTCGACGACAGTTACGATTACGTCAGGACAATAGCATAAAAGAATCGTAGTCAGGAAACGCCCCTTCTTCTTATTTAGGCACGATCCTCGCG
TTTGAGCGCAGCACCGTGGACGTGTGTCAGCTCACGGGATGGAGTGAGAGGCCCTATTGTCTGGTTACCAAAGCGCTTTTCCCGGATGTCCAAATTTTACAACGCTAAGTAACCGTCCCCGTCGTACGACTAGAGTAGAGGAG
CCG
GATCTCTG
CACGACAAAAAGATAACTCGGTCGT
CACTCGTTCTACAACATCTTCCTGTCGATCGACATGGGCTGGCGGGGCAGCCTAAGCGTATCAACTGACATTGACATAGTGTACTATAGGACAACCTACAACAGCCGACATTTAAGAAGTGGAGGGACGCGGGGAATGACTAAGACCACTTT
AGAGGGATAGGGGGCAGGTGTGGATTCGGAAGGCAATAATGGCAAGCGCACAGTACGCACCATACAAAAATTGATGTACACGATCACACCCGCAGTATTCTTAGTTAGACGCTGACGCCTGGGAGATTTGATCACCTCTTTGAGACTTGAGCGACCCA
AGATAAACTGATCGTTTATGTTAAGCCTGCTGTTGTTGAGACCCTCCGCAAATCACGCAAGTGGGACCTGATTCCGGGCAGCACGCTAGTACTTTCTATGGTGCATATCCTTGTACATCTTGAGCTTATGTGCCTGATCCAAATTAAAACTTTTCA
