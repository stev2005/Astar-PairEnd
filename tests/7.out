21
37
57
51
70
54
35
62
69
0
2
10
73
78
76
